tests/_test_utils/test_auth.py

```diff
@@ -49,7 +49,7 @@ def test_keyring():
 
 def test_get_auth_from_url_and_name():
     # set start condition to prevent definitions from a test earlier
-    constants.AUTH_LOGIN_CACHE == {}
+    constants.AUTH_LOGIN_CACHE = {}
     # empty http_auth definition
```

Comment:
good catch,  is there more broken, that was a cast to boolean.

Reply:
If this is a question I am not sure I quite get it. This was the only instance of such a mistake that I saw, but I didn't specifically scan the codebase for more.

Reply:
I assume it was added because an other test had added data  to the constants.AUTH_LOGIN_CACHE in the past, which could do a conflict.

The tests seem not have to failed without the start condition. Is the test requiring an empty AUTH_LOGIN_CACHE, is the test wrong? is that line at all needed, because it had no effect.

Reply:
The test is failing on latest develop: https://github.com/Open-MSS/MSS/actions/runs/7477525278/job/20350414652#step:9:103

It passes on first try with this change: https://github.com/Open-MSS/MSS/actions/runs/7476710573/job/20347734552?pr=2146#step:9:104

So I assume that this line is required (for now at least, maybe this should also be moved to the reset fixture? That can be judged better in a later PR, when I will try to get completely rid of the retries again).